packages/frontend/src/hooks/useExplorerRoute.ts

```diff
@@ -22,7 +22,7 @@ export const getExplorerUrlFromCreateSavedChartVersion = (
         'create_saved_chart_version',
         JSON.stringify(createSavedChart),
     );
-    if (dateZoomGranularity) newParams.set('date_zoom', dateZoomGranularity);
```

Comment:
change the query argument to be consistent with the one in dashboard. We have some unconsistent query parameters in the app